### PR TITLE
Script to download Medium articles 

### PR DESCRIPTION
**Pull request by Namyalg**
This script written in Python uses BeautifulSoup library. It can be used to download Medium Articles given the URL.

In reference to - #92

### CODE REVIEW

**Comment by Neilblaze** [comment]
Thank you for your contribution @Namyalg  !

Before I approve changes to the PR, I will request you the following changes mentioned below.

- Add README (guide) along with this inside the folder same as your `<filename>`
- Shift most of the comments as instructions in the README file.
- Maintain proper [Contribution protocols](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) (ignore if you're already following it!) & [good practices](https://gist.github.com/sloria/7001839)!
- Recheck if there are any intendation errors in the script.

Rest LGTM :+1:

**Comment by Namyalg** [comment]
Yep, will do so!

**Comment by Neilblaze** [comment]
@Namyalg it'll better if you remove those .png files from the PR folder!

File: Web-Scraping/Medium_article_downloader/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The script is written in Python
+
+The Beautiful Soup library in Python enables web scraping and enables parsing though html content, which web pages are made of. Here, the same has been used.
+
+## Implementation ##
+
+The user is prompted to enter the URL of the Medium article that has to be downloaded
+
+![Image](promptURL.PNG)
```
Comment (Neilblaze) [inline Web-Scraping/Medium_article_downloader/README.md]: Delete this image!

File: Web-Scraping/Medium_article_downloader/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The contents are then stored in a file named Medium$_article$_content.txt
+
+![Image](content.PNG)
```
Comment (Neilblaze) [inline Web-Scraping/Medium_article_downloader/README.md]: & this too!

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Comment by Neilblaze** [comment]
It is highly recommended if you modify the README to text only mode, i.e. the instructions should be in text format only!

**Comment by Namyalg** [comment]
I assumed adding images will improve the readability. I will make the
necessary changes.

**Comment by Namyalg** [comment]
I have made the necessary changes. I would like to know if I have to change anything else

**Review by Neilblaze — APPROVED** [review]
LGTM :+1: 
Good job @Namyalg :tada:

**Pull request merged**